src/Component/Region.js

```diff
@@ -0,0 +1,59 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useParams } from 'react-router';
+import { getRegion } from '../Redux/asyncActions';
+import Loading from './Loading';
+import Header from './Header';
+
+const Region = () => {
+  const region = useSelector((state) => state.region.data);
+  const loading = useSelector((state) => state.region.loading);
+  const { country } = useParams();
+  const dispatch = useDispatch();
+  useEffect(() => {
+    const abort = new AbortController();
+    dispatch(getRegion(country, abort));
+    return () => abort.abort();
+  }, []);
+  if (!loading || loading === 'loading') return <Loading />;
+  return (
+    <>
+      <Header page="region" country={country} />
+      <section className="hero px-8 bg-[#5787E5] text-white h-[15rem] flex justify-center items-center gap-8">
+        <i className="fas fa-map-marked-alt fa-6x text-[#2D4573]" />
+        <div className="CountryStats">
+          <h2 className="text-4xl font-bold text-center">{region.name}</h2>
+          <p className="text-lg text-center">
+            {loading === 'success' && `${region.today_confirmed} Cases`}
+          </p>
+        </div>
+      </section>
+      <section className="countriesStats">
+        <h2 className="bg-[#35548B] py-[4px] px-4 text-white font-medium">
+          Stats by region
+        </h2>
+        <ul className="grid grid-cols-2 justify-items-end text-right bg-[#4972be]">
+          {region.regions !== undefined && region.regions.map((region) => (
+            <li
+              key={region.id}
+              className="country w-full grid p-4 text-white"
+              data-testid="regionListItem"
+            >
+              <i className="far fa-arrow-alt-circle-right fa-lg" />
+              <i className="fas fa-map-marker-alt fa-5x text-[#2D4573] justify-self-center" />
+              <h2 className=" text-xl font-bold">{region.name}</h2>
+              <p>{`${region.today_confirmed} Cases`}</p>
+            </li>
+          ))}
+        </ul>
+        {region.regions !== undefined && region.regions.length === 0 && (
+          <div className="h-[calc(100vh-20rem)] bg-[#5787E5] text-white text-center pt-20 text-2xl font-bold">
+            No Regional Data
+          </div>
+        )}
+      </section>
+    </>
+  );
+};
+
+export default Region;
```

Comment:
- [ ] Following the project requirements, you should follow the [design guideline](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) for the detail page. As far as I can tell, you're using the same design for the detail page as you are for the home page. Take a look at the following comparison:
- Original design:
<img width="307" alt="image" src="https://user-images.githubusercontent.com/77190203/165883978-450ed637-cbcd-494d-9eba-0d322a6eaa97.png">

- Your detail page design:
<img width="206" alt="image" src="https://user-images.githubusercontent.com/77190203/165884042-06856f94-dd0d-44ca-8a4b-5d9ace428715.png">